.github/workflows/update.yml

```
@@ -0,0 +1,38 @@
+name: Update README and LaTeX Build File
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-and-create-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12.2'
+      
+      - name: Run update script
+        run: python scripts/update_files.py
+      
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+      
+      - name: Commit changes to a new branch
+        run: |
+          git checkout -b changes/${{ github.run_id }}
+          git add .
+          git commit -m "Apply automated updates"
+          git push -u origin changes/${{ github.run_id }}
+      
+      - name: Create Pull Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh pr create --base main --head changes/${{ github.run_id }} --title "Review automated updates" --body "Please review the changes applied by the automated script."
```

```suggestion
          gh pr create --base main --head changes/${{ github.run_id }} --title "Specify project URLs and CMake project" --body "Please review the changes applied by the automated script."
```

The title should communicate what changed instead of what you expect from the user.